clients/python-wrapper/lakefs/config.py

```diff
@@ -28,9 +32,15 @@ class ClientConfig:
     """
 
     class Server(NamedTuple):
+        """
+        lakectl configuration's server block
```

Comment:
This is explicitly the lakectl configuration - per the design document: https://github.com/treeverse/lakeFS/blob/12a2060812a0ca07449b73bae23cd38b846dbdde/design/accepted/high-level-python-wrapper.md?plain=1#L22

Reply:
Agree, I think it is a good fallback - so why for all the test we don't use the first or second option (init or env)? asking because of manage and create temp configuration.

If it is just for a single test - please ignore my comment.

Reply:
I can change the default to explicit params. I just reused a context that already existed